docs/kibana/reporting.md

```diff
@@ -39,3 +39,13 @@ Definitions let you schedule reports for periodic creation.
    For scheduled reports, choose either **Recurring** or **Cron based**. You can receive reports daily or at some other time interval. Cron expressions give you even more flexiblity. See [Cron expression reference](../../alerting/cron/) for more information.
 
 1. Choose **Create**.
+
+## Troubleshooting
```

Comment:
Hey @ashwinkumar12345, this looks like more of an installation issue than an issue you'd encounter when using the plugin. Would a user encounter this when they attempt to start Kibana (`./bin/kibana`)? If so, should we name this section "Chromium fails to launch with Kibana" and give the condition under which they'd encounter it (plus a formal error string?).

Reply:
It's when the user tries to generate a report and sees a download error. I added the error message.

Reply:
:thumbsup: